### DISCUSSION TITLE
Enabling Strelka-ui as upstream Server under nginx reverse proxy

### DISCUSSION THREAD

**Discussion by deepakts2013** [Security-Onion-Solutions/securityonion#12625] [Unsupported Versions]
I am trying to add Strelka UI under nginx reverse proxy . 
Here is my configuration on nginx.conf

 location   /strelka/ {
               proxy_pass http://ui:8080/ui;
               proxy_hide_header Access-Control-Allow-Origin;
               add_header Access-Control-Allow-Origin "*" always;
               proxy_set_header Host $host;
               proxy_set_header X-Real-IP $remote_addr;
               proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
               proxy_set_header X-Forwarded-Proto $scheme;
               proxy_set_header Connection "Keep-Alive";
             proxy_set_header Proxy-Connection "Keep-Alive";
              proxy_read_timeout 90;
             proxy_buffering off;
}
I see only http://<ip>/strelka/  is accessed this login page has successive call to js files which do not succeed as it points to 
http://<ip>/some.js  which do not exits it exists at http://<ip>:8080/some.js

**Comment by reyesj2** [comment]
Can I ask what you're looking to do by directly accessing Strelka?
